packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -679,7 +676,7 @@ export default function<T, P, I, TI, HI, PI, C, CC, CX, PL>(
         }
       } else {
         try {
-          commitAllLifeCycles(root, currentTime, committedExpirationTime);
```

Comment:
Why did you remove `root` here, but `null` above?

Reply:
Hey, thanks for taking some time to review the PR. As I'm looking back, I'm not sure why I removed null above, root should've been removed instead. Great catch, I've updated the PR.

Thanks!